doc/sphinx/practical-tools/utilities.rst

```diff
@@ -310,6 +316,16 @@ file timing data:
        (which are frequently noise); lexicographic sorting forces an order on
        files which take effectively no time to compile.
 
+       If you prefer a different sorting order, you can pass
+       ``TIMING_SORT_BY=absolute`` to sort by the total time taken, or
+       ``TIMING_SORT_BY=diff`` to sort by the signed difference in
+       time.
+
+    .. note::
+       Just like ``pretty-timed``, this table defaults to using user
+       times and supports ``TIMING_REAL=1`` to specify that real times
```

Comment:
```suggestion
       times by default.  Use ``TIMING_REAL=1`` on the command line to show real times instead.
```

Reply:
"defaults to using user times by default" seems worse.  What are you trying to accomplish with this suggestion?

Reply:
```suggestion
       times.  Pass ``TIMING_REAL=1`` to ``make`` on the command line to show real times instead.
```

Going to go with something slightly more informative.